src/ape_console/_cli.py

```diff
@@ -1,23 +1,23 @@
 import faulthandler
 
-import click
 import IPython  # type: ignore
 
-from ape import networks
 from ape import project as default_project
-from ape.cli import network_option, verbose_option
+from ape.cli import network_bound_command, network_option, verbose_option
 from ape.version import version as ape_version  # type: ignore
 
 
-@click.command(short_help="Load the console", context_settings=dict(ignore_unknown_options=True))
+@network_bound_command(
+    short_help="Load the console",
+    context_settings=dict(ignore_unknown_options=True),
+)
 @verbose_option(help="Display more information in the console")
 @network_option
 def cli(verbose, network):
     """
     Opens a console for the local project."""
 
-    with networks.parse_network_choice(network):
```

Comment:
Now, plugin writers can just use the network_bound_command + the network_option, and they can write their command logic.

Reply:
Two comments:
1. Would `network_option` work without `network_bound_command`?
2. Vice versa (would `network_bound_command` work without `network_option`?)

If the answer to both is "no", would it make sense to combine them into just `network_bound_command`?

Reply:
`network_bound_command` now get the `network_option` for free.
However, you can still use `network_option` without it being a network bound command, if it makes sense.